urdf/ur_macro.xacro

```diff
@@ -75,6 +75,9 @@
       visual_parameters_file="${visual_parameters_file}"
       force_abs_paths="${force_abs_paths}"/>
 
+    <xacro:property name="config_joint_limit_parameters" value="${xacro.load_yaml(joint_limits_parameters_file)}"/>
+    <xacro:property name="sec_limits" value="${config_joint_limit_parameters['joint_limits']}"/>
```

Comment:
We can probably get around this by putting the parsed values into the parent scope in `ur_common.xacro`